### ISSUE TITLE
Several pages have controls that are not readable in dark mode

### ISSUE THREAD

**Issue by marlenecota** [microsoft/react-native-gallery#456]
- [ ] Button
![image](https://github.com/microsoft/react-native-gallery/assets/1422161/f59cfad7-7532-4598-bbd5-652603e9982f)
- [ ] Pressable
![image](https://github.com/microsoft/react-native-gallery/assets/1422161/ca97bf62-f5e8-4bc4-8f87-7c983c69aadf)
![image](https://github.com/microsoft/react-native-gallery/assets/1422161/eeb8f664-b9b0-45a2-b95d-4fa33c1a894b)
- [ ] Switch
![image](https://github.com/microsoft/react-native-gallery/assets/1422161/62d5a1a3-cdc0-4e70-8a67-232932ce20e8)
- [ ] Flyout
![image](https://github.com/microsoft/react-native-gallery/assets/1422161/3e9569c8-0367-440e-9692-04532862e24e)
![image](https://github.com/microsoft/react-native-gallery/assets/1422161/712ca8c8-ae21-4128-9763-8d3506501a25)
![image](https://github.com/microsoft/react-native-gallery/assets/1422161/0d6a7036-5901-4eac-8742-b21f15680e6b)
![image](https://github.com/microsoft/react-native-gallery/assets/1422161/babf622a-2e20-4f15-bb27-1a148453d021)
- [ ] TouchableHighlight
![image](https://github.com/microsoft/react-native-gallery/assets/1422161/4819a36a-d61f-49e6-9e68-e6b4eda76b92)
![image](https://github.com/microsoft/react-native-gallery/assets/1422161/c04f3160-1f0c-4e91-8b7c-74cf059eea85)
- [ ] TouchableOpacity
![image](https://github.com/microsoft/react-native-gallery/assets/1422161/eb32ee2d-3f09-42c0-93be-5762df6a3224)
![image](https://github.com/microsoft/react-native-gallery/assets/1422161/b4f3b301-2b6f-47d3-8b50-65f16f72a79d)
![image](https://github.com/microsoft/react-native-gallery/assets/1422161/fb971efd-c2a2-4d8c-a090-538a2226a448)
![image](https://github.com/microsoft/react-native-gallery/assets/1422161/b7db5090-26b3-4631-98de-75ec9132e7a6)

**Comment by chrisglein** [comment]
Detail from Marlene: Switch looked fine on hover. So something specific to idle colors is off?
Definitely the colors were touched as part of visual rejuvenation pass, not sure if something regressed or was broken before.

**Comment by chrisglein** [comment]
Recently had an accessibility pass on the app. Some of this may be addressed as part of high contrast fixes.

**Comment by chrisglein** [comment]
@Yajur-Grover Can you confirm that these are fixed with your high contrast fix pass?
